tests/stub/driver_execute_query/scripts/tx_return_1_with_re_auth.script

```diff
@@ -0,0 +1,26 @@
+!: BOLT 5.1
+
+A: HELLO {"user_agent": "*", "[routing]": "*"}
+A: LOGON {"scheme": "basic", "principal": "neo4j", "credentials": "pass", "[realm]": ""}
+*: RESET
+{+
+    C: ROUTE "*" "*" "*"
+    S: SUCCESS {"rt": {"ttl": 1000, "db": "adb", "servers": [{"addresses": ["#HOST#:9000"], "role":"ROUTE"}, {"addresses": ["#HOST#:9010"], "role":"READ"}, {"addresses": ["#HOST#:9020"], "role":"WRITE"}]}}
+    ?: RESET
++}
```

Comment:
This script is only ever played with a non-routing driver. Why do we expect and accept a `ROUTE` request?

Reply:
Thanks for pairing on this, script is now fixed 👍

Reply:
This should've failed :grimacing: 
Thanks for finding a bug in the stub server https://github.com/neo4j-drivers/testkit/pull/613